### PR TITLE
Use FB named pools. Duplicate for VS2019. Use version number in path.

### PR DESCRIPTION
**Pull request by drodriguez**
@compnerd I haven't changed the current agent from "Default" to "Facebook-VS2017", to avoid breaking the current behaviour. If you want to merge this, I can bring down the agent and register it again in the right pool.

### CODE REVIEW

File: .ci/toolchain-vs2019-amd64-facebook.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - .ci/azure-pipelines-aws-agent.yml
```
Comment (compnerd) [inline .ci/toolchain-vs2019-amd64-facebook.yml]: This should be `.ci/toolchain-vs2019-amd64-facebook.yml`.

File: .ci/toolchain-vs2019-amd64-facebook.yml [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+      - .ci/templates/toolchain.yml
+trigger:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - .ci/azure-pipelines-aws-agent.yml
```
Comment (compnerd) [inline .ci/toolchain-vs2019-amd64-facebook.yml]: Similar

File: .ci/toolchain-vs2019-amd64-facebook.yml [hunk 3]
```diff
@@ -0,0 +1,79 @@
+pr:
+  branches:
+    include:
+      - master
+  paths:
+    include:
+      - .ci/azure-pipelines-aws-agent.yml
+      - .ci/templates/toolchain.yml
+trigger:
```
Comment (compnerd) [inline .ci/toolchain-vs2019-amd64-facebook.yml]: Do we want to do a `schedules` trigger with `*/90 * * * *` to trigger this every 90 minutes?

**Comment by drodriguez** [comment]
There seems to be one running at the moment (https://dev.azure.com/compnerd/windows-swift/_build/results?buildId=18011&view=results), so maybe in 20 minutes?

The other agent is already in the right pool. When this is merge, new pipelines should be created pointing to these new configurations. When we get things rolling, we can set crons for them.

**Comment by drodriguez** [comment]
One agent in each pool. Merge at your earliest convenience.

**Comment by compnerd** [comment]
Merged.

**Comment by compnerd** [comment]
Adjusted the existing 2017 pipeline, adding the 2019.

**Comment by drodriguez** [comment]
The machine was missing touching a file for Python 2.7 to work. Done and started the pipeline in https://dev.azure.com/compnerd/windows-swift/_build/results?buildId=18054&view=results

**Pull request closed without merging**